docs/category/category-schema.yaml

```diff
@@ -18,9 +18,16 @@ definitions:
         subjectsIcon:
           type: string
         totalOffers:
-          type: number
-          default: 0
-          $ref: '#/definitions/offer'
+          type: object
+          properties:
+            student:
+              type: number
+              default: 0
+              $ref: '#/definitions/offer'
```

Comment:
i'm not sure that $ref field here is needed, cause student in totalOffers is a number in response but not offer

Reply:
You are right. I just copy it and I forgot to delete it. My bad.